docs/_dg/implementations/TagFeatures.md

```diff
@@ -49,7 +49,7 @@ When storing a tag, these are the alternatives considered.
     * Easily extensible (defining a new `Tag` or `Item` can be as simple as creating a new node in the tree)
 
   * Cons:
-    * Example of over engineering, unnecessarily complicates things as compared to alternative 1
+    * Example of over-engineering, unnecessarily complicating things, as compared to alternative 1
```

Comment:
Phrasing is a bit weird. Thoughts on the following?

```suggestion
    * Example of over-engineering, unnecessarily complicates things when compared to alternative 1
```

Reply:
Yep sounds good